CMakeLists.txt

```diff
@@ -149,6 +149,7 @@ set ( OPENXCOM_VERSION_GIT "${GIT_STATE}" CACHE STRING "Git version string (afte
 
 configure_file("${CMAKE_SOURCE_DIR}/src/git_version.h.in" "${CMAKE_CURRENT_BINARY_DIR}/git_version.h" )
 include_directories ( "${CMAKE_CURRENT_BINARY_DIR}" )
+include_directories ( ${DEPS_DIR}/entt/single_include )
```

Comment:
this line isn't needed anymore since the entt system is included in the src/CMakeLists.txt now